solutions/swb-reference/integration-tests/tests/isolated/projects/removeUserFromProject.test.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+import { resourceTypeToKey } from '@aws/workbench-core-base';
+import ClientSession from '../../../support/clientSession';
+import Setup from '../../../support/setup';
+import HttpError from '../../../support/utils/HttpError';
+import { checkHttpError } from '../../../support/utils/utilities';
+
+describe('remove user from project negative tests', () => {
+  const setup: Setup = new Setup();
+  let adminSession: ClientSession;
+
+  beforeEach(() => {
+    expect.hasAssertions();
+  });
+
+  beforeAll(async () => {
+    adminSession = await setup.getDefaultAdminSession();
+  });
+
+  afterAll(async () => {
+    await setup.cleanup();
+  });
+
+  test('user does not exist', async () => {
```

Comment:
are we planning a multi step later to create the user and test the removal? or we are going to always testing the non existing user ?